.docker/Dockerfile

```diff
@@ -1,14 +1,73 @@
-FROM rocketchat/base:12.16.1
+FROM debian:buster-slim
 
 ADD . /app
```

Comment:
Shouldn’t this happen well after the nodejs part to increase caching?

Reply:
yes, definitely